gdc_rnaseq_tools/augment_star_counts.py

```diff
@@ -0,0 +1,228 @@
+import numpy as np
+import pandas as pd
```

Comment:
You don't have to change anything, but I don't usually recommend the use of `pandas` since for most applications `numpy` is sufficient.

For example, for this code I would read each table into a dictionary of `{ colname: np.array(column_data) }`.
The `merge` operation would be slightly tricker but not impossible.